src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Enable mocking of `final` methods, necessary because `Run#getStartTimeInMillis()` is used by the `GlobalAnnotator` during a test and I'd like to provide it with a mock `Run` instead of a real dummied-up `Run`.